tgui/packages/tgui/interfaces/ExaminePanel.jsx

```diff
@@ -50,165 +58,253 @@ export const ExaminePanel = (props) => {
+                      minHeight="50%"
+                      maxHeight="50%"
+                      fontSize="14px"
+                      lineHeight="1.7"
+                      textIndent="3em"
+                    >
+                      {formatURLs(flavor_text_nsfw)}
+                    </Section>
+                  )}
+                  {tabIndex === 3 && (
+                    <Section
+                      style={{ 'overflow-y': 'scroll' }}
+                      fitted
+                      preserveWhitespace
+                      minHeight="50%"
+                      maxHeight="50%"
+                      fontSize="14px"
+                      lineHeight="1.7"
+                      textIndent="3em"
+                    >
+                      {custom_species
+                        ? formatURLs(custom_species_lore)
+                        : 'Just a normal space dweller.'}
+                    </Section>
+                  )}
+                  <Tabs fluid>
+                    <Tabs.Tab
+                      selected={lowerTabIndex === 1}
+                      onClick={() => setLowerTabIndex(1)}
+                    >
+                      <Section
+                        fitted
+                        title={'OOC Notes'}
+                        style={{ textAlign: 'center' }}
+                      />
+                    </Tabs.Tab>
+                    <Tabs.Tab
+                      selected={lowerTabIndex === 2}
+                      onClick={() => setLowerTabIndex(2)}
+                    >
+                      <Section
+                        fitted
+                        title={'Character Advert'}
+                        style={{ textAlign: 'center' }}
+                      />
+                    </Tabs.Tab>
+                  </Tabs>
+                  {lowerTabIndex === 1 && (
+                    <Section
+                      style={{ 'overflow-y': 'scroll' }}
+                      preserveWhitespace
+                      fitted
+                      minHeight="35%"
+                      maxHeight="35%"
+                      fontSize="14px"
+                      lineHeight="1.5"
+                    >
+                      <Stack.Item>{formatURLs(ooc_notes)}</Stack.Item>
+                    </Section>
+                  )}
+                  {lowerTabIndex === 2 && (
+                    <Section
+                      style={{ 'overflow-y': 'scroll' }}
+                      preserveWhitespace
+                      fitted
+                      minHeight="35%"
+                      maxHeight="35%"
+                      fontSize="14px"
+                      lineHeight="1.5"
+                    >
+                      <Stack.Item>{formatURLs(character_ad)}</Stack.Item>
+                    </Section>
+                  )}
+                </Box>
+              </Stack.Item>
+            </Stack>
+          </>
+        ) : (
+          <div>
+            <Button
```

Comment:
'content' is deprecated, put it as a children of <Button>
that is, <Button> childrenGoHere </Button>